subcommittees/proposal_instructions.md

```diff
@@ -0,0 +1,35 @@
+##Software Carpentry Subcommittee & Task Force Program
+
+One of the greatest things about Software Carpentry is the huge diversity of people and ideas that have come together to make this project what it is: a worldwide collaboration of folks working together to build software skills. Your contributions to the community have made Software Carpentry a huge success - but we can go even further. To meet the needs of the growing global community, the 2016 Steering Committee is creating the opportunity for you to make Software Carpentry what your community needs it to be, via our **newly revamped Subcommittee & Task Force Program**.
```

Comment:
This “One of the greatest things…” wording is similar to, but not the same as, the new subcommittees/README.md phrasing.  I'd just dump this whole paragraph and expect folks will have read the README.  The README also includes a lot of the same advice (e.g. specificity).  Maybe these are really trying to be a single document?  I'd suggest:

* `README.md`
  * Purpose and Rationale
    * Keep current content
  * Current Subcommittees and Task Forces
    * Keep current content
  * How to Propose a New Subcommittee or Task Force
    * Replace with short blurb (like the current “Concluding Remarks” phrasing?) that links to `proposal_instructions.md`.
  * Concluding Remarks
    * Drop this, since the initial “Purpose and Rationale” paragraph already covers the “we're great” angle, and the “How to Propose…” wording will cover “we'd like to hear from you” ;).
* `proposal_instructions.md`
  * Drop initial paragraph, possibly replacing it with something that lets you link the README
  * Pull in more detailed advice from the current README's “How to Propose…”